app-admin/fluent-bit/fluent-bit-4.0.3.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Fast Log processor and Forwarder"
+HOMEPAGE="https://fluentbit.io/"
+SRC_URI="https://github.com/fluent/fluent-bit/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+
+IUSE="geoip kafka lua"
+
+DEPEND="
+	acct-group/fluentd
+	acct-user/fluentd
+	app-arch/zstd:=
+	dev-libs/libyaml
+	dev-libs/openssl:=
+	net-dns/c-ares:=
+	net-libs/nghttp2
+	sys-libs/zlib
+	kafka? ( dev-libs/librdkafka )
+	lua? ( dev-lang/luajit )
+	geoip? ( dev-libs/libmaxminddb )
+"
+RDEPEND="${DEPEND}"
+BDEPEND="
+	sys-devel/bison
+	sys-devel/flex
+"
+
+PATCHES=(
+	"${FILESDIR}/fluent-bit-4.0.3-wasm-cmake-version.patch"
+	"${FILESDIR}/fluent-bit-4.0.3-external-libmaxminddb.patch"
+	"${FILESDIR}/fluent-bit-4.0.3-wasm-noexec-stack.patch"
```

Comment:
Are any of these patches merged already upstream? If yes, you should consider packaging a snapshot version instead based on a working git commit.

Reply:
https://github.com/fluent/fluent-bit/pull/10514

But I'd rather have them take it through their release processes, don't want to deviate too far from the released versions.

The cmake version relates to the QA issue, which is still cropping up, so could use help there.

The libmaxmind they simply won't take - it's basically a hard-hack and would need to be a more comprehensive patch for them to accept it upstream.  Don't feel comfortable enough with CMake (yet) to do that currently.